labels.yaml

```diff
@@ -0,0 +1,284 @@
+# Standard labels set (common for all kubernetes repos)
+---
+labels:
+  - color: 15dd18
+    name: lgtm
+  - color: 006b75
+    name: for-new-contributors
+  - color: 006b75
+    name: help-wanted
+  - color: d2b48c
+    name: sig/api-machinery
+  - color: d2b48c
+    name: sig/apps
+  - color: d2b48c
+    name: sig/auth
+  - color: d2b48c
+    name: sig/autoscaling
+  - color: d2b48c
+    name: sig/aws
+  - color: d2b48c
+    name: sig/big-data
+  - color: d2b48c
+    name: sig/cli
+  - color: d2b48c
+    name: sig/cluster-lifecycle
+  - color: d2b48c
+    name: sig/cluster-ops
+  - color: d2b48c
+    name: sig/contributor-experience
+  - color: d2b48c
+    name: sig/docs
+  - color: d2b48c
+    name: sig/federation
+  - color: d2b48c
+    name: sig/instrumentation
+  - color: d2b48c
+    name: sig/network
+  - color: d2b48c
+    name: sig/node
+  - color: d2b48c
+    name: sig/onprem
+  - color: d2b48c
+    name: sig/openstack
+  - color: d2b48c
+    name: sig/rktnetes
+  - color: d2b48c
+    name: sig/scalability
+  - color: d2b48c
+    name: sig/scheduling
+  - color: d2b48c
+    name: sig/service-catalog
+  - color: d2b48c
+    name: sig/storage
+  - color: d2b48c
+    name: sig/testing
+  - color: d2b48c
+    name: sig/windows
+  - color: c7def8
+    name: kind/api-change
+  - color: e11d21
+    name: kind/bug
+  - color: c7def8
+    name: kind/cleanup
+  - color: c7def8
+    name: kind/design
+  - color: c7def8
+    name: kind/documentation
+  - color: c7def8
+    name: kind/enhancement
+  - color: c7def8
+    name: kind/feature
+  - color: f7c6c7
+    name: kind/flake
+  - color: c7def8
+    name: kind/friction
+  - color: f7c6c7
+    name: kind/mesos-flake
+  - color: c7def8
+    name: kind/new-api
+  - color: c7def8
+    name: kind/old-docs
+  - color: bfe5bf
+    name: kind/postmortem
+  - color: eb6420
+    name: kind/support
+  - color: c7def8
+    name: kind/technical-debt
+  - color: fbca04
+    name: kind/upgrade-test-failure
+  - color: 0052cc
+    name: area/admin
```

Comment:
I don't know which ones are interesting and which ones aren't. I've just added lables requested.
Actually labels.yaml will probably be a lot shorter for a start. And `repos.yaml` file will be used instead of getting all kubernetes repos via GitHub call (because we want to sync only few repos for a start). so those two files: `labels.yaml` and `repos.yaml` will have to be tweaked to some wide accepted consensus.